charts/aws-vpc-cni/values.yaml

```diff
@@ -88,6 +88,25 @@ env:
   VPC_CNI_VERSION: "v1.18.2"
   NETWORK_POLICY_ENFORCING_MODE: "standard"
 
+# Add env from configMap or from secrets
+# - name: ENV_VAR1
+#   valueFrom:
+#     configMapKeyRef:
+#       name: example-config
+#       key: ENV_VAR1
+# - name: ENV_VAR2
+#   valueFrom:
+#     configMapKeyRef:
+#       name: example-config
+#       key: ENV_VAR2
+# - name: SECRET_VAR1
+#   valueFrom:
+#     secretKeyRef:
+#       name: example-secret
+#       key: SECRET_VAR1
+extraEnv: []
```

Comment:
Please move the above example comment below line 107.
You could keep it as single example instead 3, and comment it as
> Optional: Additional environment variables for helm chart.

Reply:
Also, as I understand, this is helm chart only feature. This wont be supported by self managed addon (applying yaml directly, in which case, they will have to edit the environment variables directly and with MAO.

The cost of maintaining this difference needs to be considered.


If it is strictly not required, I would consider dropping this, if any other workaround is available with helm chart.